pkgs/by-name/no/notion-app/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  unzip,
+  testers,
+  libplist,
+  jq,
+  writeText,
+  runCommand,
+}:
+let
+  info =
+    (builtins.fromJSON (builtins.readFile ./info.json))."${stdenvNoCC.targetPlatform.system}"
+      or (throw "notion-app: unsupported system ${stdenvNoCC.targetPlatform.system}");
+in
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "notion-app";
+
```

Comment:
No need for Coronavirus indentation for all parameters.

Reply:
Do you means I should remove empty line break?

Reply:
Yes, at least in this case. There is a lot of examples on how stuff is ordered in package definitions in the repo tho.

Rule of thumb is to follow the ordering of how stuff happens. It's normal at first to be a bit lost on these stuff but as you are dealing with more PRs you get more familiar to how stuff is handled.

Basically the following
- name and pname
- fetch
- unpack
- patch
- configure
- build
- install
- fixup
- passthru
- meta